lib/msg-v4/validation.js

```diff
@@ -332,7 +332,11 @@ function validate(msg, tangle, sigkeys, msgID, rootID) {
   if ((err = validateSigkey(msg))) return err
   if ((err = validateData(msg))) return err
 
-  if (tangle.type === 'feed' && isMoot(msg)) return // nothing else to check
+  try {
+    if (tangle.type === 'feed' && isMoot(msg)) return // nothing else to check
+  } catch (err) {
+    return err
+  }
```

Comment:
`tangle.type` is a function that sometimes (like in this test) throws. Changed it to this to be able to see the err causes